dvclive: 2.0 docs (adds make_dvcyaml) (#4286)
iterative/dvc.org: content/docs/dvclive/api-reference/live/index.md

  `Live.end()`. Ignored if your repo already has a
  [DVC pipeline](/doc/user-guide/pipelines). Defaults to `False`.

> - `dvcyaml` - If `True`, DVCLive will write

Reviewer: let's mention that it will be part of the `Live.end()` call

  `Live.end()`. Ignored if your repo already has a
  [DVC pipeline](/doc/user-guide/pipelines). Defaults to `False`.

- `dvcyaml` - If `True`, DVCLive will write
  [DVC configuration](/doc/user-guide/project-structure/dvcyaml-files) for
  metrics, plots, and parameters to
  [`Live.dvc_file`](/doc/dvclive/api-reference/live#properties). See
  `Live.make_dvcyaml()`. Defaults to `True`.

<admon type="tip">

If you have custom DVC configuration of metrics, plots, and parameters in your
own `dvc.yaml` file, set `dvcyaml=False` to avoid duplication.

> </admon>

Author: Added a tip here per https://github.com/iterative/dvclive/pull/443#pullrequestreview-1281648605

  [`Live.dvc_file`](/doc/dvclive/api-reference/live#properties). See
  `Live.make_dvcyaml()`. Defaults to `True`.

<admon type="tip">

> If you have custom DVC configuration of metrics, plots, and parameters in your
> own `dvc.yaml` file, set `dvcyaml=False` to avoid duplication.

Reviewer 2: WDYT about making it more explicit that the problem is only if there are overlaps? 

```suggestion
If you have are already tracking DVCLive outputs (metrics, plots, and parameters) in your
own `dvc.yaml` file, set `dvcyaml=False` to avoid duplication.
```

Author: ```suggestion
If you are already tracking DVCLive metrics, plots, and parameters in your
own `dvc.yaml` file, set `dvcyaml=False` to avoid duplication.
```

Updated a typo and avoided the term "outputs" since they could be top-level metrics/plots/params.
